mods/sgssb/abilities.js

```diff
@@ -230,4 +230,36 @@ exports.BattleAbilities = {
 		id: "nodebot",
 		name: "node bot",
 	},
+
+	//In Tandem with Lycanium Z's Wreck Havoc Move
+	virus: {
+		shortDesc: "Transforms the pokemon to unown and other odd effects.",
+		id: "virus",
+		name: "Virus",
+		onAfterDamage: function (damage, target, source, move) {
+			if (source && source !== target && move && move.flags['contact']) {
```

Comment:
This has the ``move.flags['contact']`` meaning it'll only work if the target makes contact with the user, might want to add that in the descriptions

Reply:
Good idea